### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by EGiataganas**
### Description of change

Improve UI following Hackney Design System

* Delete govuk-frontend from package json
* Start using hackney's default button
* Update app and document to Hackney Design system
* Replace login/logout with signin/signout
* Update search component
* Typography:Body-text, replace govuk-body to lbh-body
* Typography:Table, add lbh-table on table tag
* Typography:Caption, add lbh-caption
* Typography:Headings, update to match to Hackney Design system
* Typography:Links, use/add lbh-link class
* Typography:Fonts, apply Hackney Design sytem bold
* Typography:List, apply Hackney Design sytem list
* Colours, Use Hackney Design system colours
* Use Table component
* Button, apply Hackney Design system secondary button

### Story Link

https://www.pivotaltracker.com/story/show/178395035

### CODE REVIEW

File: src/pages/_document.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import Document, { Html, Head, Main, NextScript } from 'next/document'
 
 class AppDocument extends Document {
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx)
+    return { ...initialProps }
+  }
+
```
Comment (NeilvB) [inline src/pages/_document.js:8]: How come this was added? I couldn't see how it's related to styling changes..

**Review by NeilvB — APPROVED** [review]
The new UI is looking excellent.

Lots of careful, methodical work here.

My only question is about the spacing between the property information and the tabs that sit underneath it - see the tenure text and the tab in the image below. It looks like we may have lost spacing here somewhere. Is that intentional?

property-spacing

**Review by siapankina — APPROVED** [review]
looks great

File: src/pages/_document.js (hunk 1 above)
Comment (EGiataganas) [reply src/pages/_document.js:8]: I've compared it from Hackney Design System [examples][1]

[1]:https://github.com/LBHackney-IT/hackney-design-system-examples/blame/main/next/pages/_document.js

**Comment by EGiataganas** [comment]
Hackney Design system has a bit different spacing. For example `govuk` has `margin-bottom: 20px;` where hackney design system doesn't. In addition I've also wrapped all the links to `lbh-link` class which this has caused some spacing changing. I am in contact with Yasmeen and once we migrate fully some of the screens will changed.

**Pull request merged**